scss/_custom-forms.scss

```diff
@@ -210,6 +210,8 @@
 .custom-file-input {
   min-width: $custom-file-width;
   max-width: 100%;
+  padding-top: $custom-file-padding-y;
+  padding-bottom: $custom-file-padding-y;
```

Comment:
This padding is to make the hidden <input> element match the height of the custom control. Without this extra height, files can only be dragged onto the top half of the control.